src/dashboards/selectors/index.ts

```diff
@@ -64,11 +63,12 @@ export const isCurrentPageDashboard = (state: AppState): boolean =>
 // Example: user selected 10-11:00am and sets the dropdown to UTC
 // Query should run against 10-11:00am UTC rather than querying
 // 10-11:00am local time (offset depending on timezone)
-export const setTimeToUTC = (date: string): string =>
-  moment
-    .utc(date)
-    .subtract(getTimezoneOffset(), 'minutes')
-    .format()
+export const setTimeToUTC = (date: string): string => {
+  const utcTime = new Date(date)
+  utcTime.setUTCMinutes(utcTime.getUTCMinutes() - getTimezoneOffset())
+
+  return utcTime.toISOString().split('.')[0] + 'Z'
```

Comment:
1. rather than concatenating, which might be confusing in the context of subtracting two timestamps, it would be slightly cleaner to use template strings (`${dateString}Z`)
1. i'm sure you tested this, but just for extra confidence can you add a few jest tests around this method and then run it against master and make sure both pass

Reply:
I can add more dedicated tests! There is a test in another file that uses this function so I thought I could skip the test for this, but like you said in the standup the other day we'd like to add tests for everything, I can add that 👍